src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -92,6 +92,19 @@ interface LicenceRepository : JpaRepository<Licence, Long>, JpaSpecificationExec
     """,
   )
   fun getLicencesPassedExpiryDate(): List<Licence>
+
+  @Query(
+    """
+    SELECT l
+        FROM Licence l
+        WHERE l.statusCode  IN (
+            uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.IN_PROGRESS,
+            uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.SUBMITTED
+        )
+        AND COALESCE(l.conditionalReleaseDate, l.actualReleaseDate) < CURRENT_DATE
+    """,
+  )
+  fun getDraftLicencesPassedCRD(): List<Licence>
```

Comment:
Slight misnomer, perhaps `getDraftLicencesPassedReleaseDate` would be more accurate? Not a big deal though.

Reply:
updated to release date